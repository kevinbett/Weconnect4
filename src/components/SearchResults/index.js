import React from 'react';
import { Link } from 'react-router-dom';

const SearchResults = (props) => {
  const businesses = props.results.map(business => (
    <div className="card-deck row" key={business.id}>
    <div className="card">
      <div className="card-header bg-transparent">{business.name}</div>
      <div className="card-body">
        <p className="card-text">Category {business.category}</p>
        <p className="card-text">Description {business.type}</p>
        <p className="card-text">Location {business.location}</p>
      </div>
      <div className="card-footer">
        <Link to={`profile/${business.id}`} className="btn btn-primary btn-sm float-right">See more</Link>
      </div>
    </div>
  </div>

  ))

  return <div className="row"> <ul>{businesses}</ul></div>

}

export default SearchResults