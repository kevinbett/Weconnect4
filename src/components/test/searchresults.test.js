import SearchResults from '../SearchResults'
import { shallow } from "enzyme"

describe('search results page ',() => { 
    it ('search results page renders', () => { 
        const results = [
            {
                "category": "tech",
                "id": 3,
                "location": "trm",
                "name": "android",
                "type": "tech"
            }
        ];
        expect(shallow(<SearchResults results={ results }/>)).toMatchSnapshot()
    });
} )