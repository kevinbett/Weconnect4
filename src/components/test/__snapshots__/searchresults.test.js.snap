// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search results page  search results page renders 1`] = `
<div
  className="row"
>
   
  <ul>
    <div
      className="card-deck row"
      key="3"
    >
      <div
        className="card"
      >
        <div
          className="card-header bg-transparent"
        >
          android
        </div>
        <div
          className="card-body"
        >
          <p
            className="card-text"
          >
            Category 
            tech
          </p>
          <p
            className="card-text"
          >
            Description 
            tech
          </p>
          <p
            className="card-text"
          >
            Location 
            trm
          </p>
        </div>
        <div
          className="card-footer"
        >
          <Link
            className="btn btn-primary btn-sm float-right"
            replace={false}
            to="profile/3"
          >
            See more
          </Link>
        </div>
      </div>
    </div>
  </ul>
</div>
`;
